### PR TITLE
chore(linux): Fix lintian warnings

### PR DESCRIPTION
**Pull request by ermshiperete**
See https://mentors.debian.net/package/keyman/ for Lintian warnings.

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by mcdurdin** [comment]
Lintian warnings from that page (since it will change in future):

```
```
keyman source
E [not-binnmuable-all-depends-any](https://lintian.debian.org/tags/not-binnmuable-all-depends-any.html)
keyman -> ibus-keyman
E [source-is-missing](https://lintian.debian.org/tags/source-is-missing.html)
common/core/desktop/src/ibus_keyman_tests-json.o
common/core/desktop/src/ibus_keyman_tests-keyboard.o
common/core/desktop/src/ibus_keyman_tests-option.o
common/core/desktop/src/ibus_keyman_tests-state.o
common/core/desktop/src/ibus_keyman_tests-utfcodec.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_actions.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_capslock.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_consts.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_context.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_conversion.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_debug.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_debugger.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_environment.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_file.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_modifiers.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_options.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_processevent.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_processor.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_xstring.o
common/core/desktop/tests/unit/kmx/ibus_keyman_tests-kmx_test_source.o
I [out-of-date-standards-version](https://lintian.debian.org/tags/out-of-date-standards-version.html)
4.5.1 (released 2020-11-17) (current is 4.6.0.1)
I [override_dh_auto_test-does-not-check-DEB_BUILD_OPTIONS](https://lintian.debian.org/tags/override_dh_auto_test-does-not-check-DEB_BUILD_OPTIONS.html)
(line 46)
P [maintainer-desktop-entry](https://lintian.debian.org/tags/maintainer-desktop-entry.html)
debian/km-config.desktop
P [maintainer-manual-page](https://lintian.debian.org/tags/maintainer-manual-page.html)
debian/man/ibus-engine-keyman.1
P [package-uses-old-debhelper-compat-version](https://lintian.debian.org/tags/package-uses-old-debhelper-compat-version.html)
11
P [silent-on-rules-requiring-root](https://lintian.debian.org/tags/silent-on-rules-requiring-root.html)
P [source-contains-prebuilt-binary](https://lintian.debian.org/tags/source-contains-prebuilt-binary.html)
common/core/desktop/src/ibus_keyman_tests-json.o
common/core/desktop/src/ibus_keyman_tests-keyboard.o
common/core/desktop/src/ibus_keyman_tests-option.o
common/core/desktop/src/ibus_keyman_tests-state.o
common/core/desktop/src/ibus_keyman_tests-utfcodec.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_actions.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_capslock.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_consts.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_context.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_conversion.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_debug.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_debugger.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_environment.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_file.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_modifiers.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_options.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_processevent.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_processor.o
common/core/desktop/src/kmx/ibus_keyman_tests-kmx_xstring.o
common/core/desktop/tests/unit/kmx/ibus_keyman_tests-kmx_test_source.o
P [uses-debhelper-compat-file](https://lintian.debian.org/tags/uses-debhelper-compat-file.html)
P [very-long-line-length-in-source-file](https://lintian.debian.org/tags/very-long-line-length-in-source-file.html)
CONTRIBUTING.md line 118 is 543 characters long (>512)
linux/ibus-keyman/autom4te.cache/output.0 line 14273 is 714 characters long (>512)
linux/ibus-keyman/autom4te.cache/output.1 line 14273 is 714 characters long (>512)
linux/ibus-keyman/autom4te.cache/output.2 line 14273 is 714 characters long (>512)
linux/ibus-keyman/autom4te.cache/traces.0 line 5373 is 603 characters long (>512)
linux/ibus-keyman/autom4te.cache/traces.2 line 5373 is 603 characters long (>512)
linux/ibus-keyman/configure line 14273 is 714 characters long (>512)
linux/ibus-keyman/m4/libtool.m4 line 6620 is 738 characters long (>512)
linux/ibus-keyman/m4/po.m4 line 410 is 603 characters long (>512)
linux/ibus-keyman/tests/Makefile.in line 1424 is 689 characters long (>512)
linux/keyman-config/keyman_config/standards/lang_tags_map.py line 9069 is 654357 characters long (>512)
resources/standards-data/langtags/langtags.json line 32666 is 970 characters long (>512)
```

File: .editorconfig [hunk 1]
```diff
@@ -9,7 +9,8 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.md]
-max_line_length = off
+# Lintian in Debian complains about lines > 512
+max_line_length = 512
```
Comment (mcdurdin) [inline .editorconfig:13]: I think I'd prefer .md to be wrapping at 80 chars. This is what I am using everywhere I edit .md files -- it's sometimes a problem for URLs so we don't enforce that length.

**Review by mcdurdin — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (ermshiperete) [reply .editorconfig:13]: I agree, but I'm not sure what do you suggest to do here?

Comment (mcdurdin) [reply .editorconfig:13]: We could set to 160 chars max length? That's less likely to cause trouble.

Comment (ermshiperete) [reply .editorconfig:13]: Changed in #6378.

**Pull request merged**